File: V_1/TravkingApplicationAPI/DTO/AddTask.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using TravkingApplicationAPI.Models;

namespace TravkingApplicationAPI.DTO
{
    public class AddTask
    {
        public int UserTaskID { get; set; }
    public string TaskName { get; set; }
    public string Description { get; set; }
    public priority Priority { get; set; }
    public DateTime DeadLine { get; set; }
    public int Status { get; set; }
    public int AssignedBy { get; set; }
    public List<int> AssignedTo { get; set; }
    public int BatchId { get; set; }
    public string Comments { get; set; }

    }
}

File: V_1/TrackigApplicationFrontend/src/app/Models/add-task.ts
export class AddTask {

    userTaskID!: number;
    TaskName!: string;
    Description!: string;
    Priority!: priority;
    DeadLine!: Date;
    Status!: number;
    AssignedBy!: number;
    AssignedTo!: number;
    BatchId!: number;
    Comments!: Comment;
    ModuleId!:number;


}
export enum priority{

  low
  ,medium,
  high
}


